torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3830,6 +3830,48 @@ def sample_inputs_matmul(op_info, device, dtype, requires_grad):
     return tuple(sample_inputs)
 
 
+def sample_inputs_meshgrid(
+        calling_convention: str) -> Callable[[OpInfo, torch.device, torch.dtype, bool], List[SampleInput]]:
```

Comment:
Oh wow... I wouldn't type any of this unless you have to (and even then in the future I'd just tell the linter to ignore typing issues)

Reply:
(but what's done is done and there's no reason to remove it as long as it works)

Reply:
it wasn't a burden, and it helped me to understand what the actual types were. it could be argued that adding all these types clutters the code and impedes readability. would you say that?